### DISCUSSION TITLE
[Translation] - Monika After Story 0.11.9 translation into Simplified Chinese

### DISCUSSION THREAD

**Discussion by mtr-static-official** [Monika-After-Story/MonikaModDev#7828] [general - translations]
Monika After Story translation into Simplified Chinese.
Version: 0.11.9
Translation team list: plz see readme
Link to the build of the translation:
DDLC Discussion Board: 
https://forum.monika.love/d/234-mas-monika-after-story-0119
MonikaCloud:
https://cloud.monika.love/#/s/dGZIy

Monika After Story简体中文版.
版本:0.11.9
汉化名单:见README文档
DDLC Discussion Board: 
https://forum.monika.love/d/234-mas-monika-after-story-0119
MonikaCloud:
https://cloud.monika.love/#/s/dGZIy
#5691

**Comment by 8MiYile** [comment]
抱歉 主下载服务并不可用，请留一个mirror地址 感谢

**Comment by mtr-static-official** [comment]
[https://pan.monika.love/s/dGZIy](https://pan.monika.love/s/dGZIy) @8MiYile

**Comment by 8MiYile** [comment]
@mtr-static-official thanks

**Comment by Cintser** [comment]
抱歉我不知道该如何下载翻译包，似乎云盘等也接连挂掉
